src/convertSourceToRaw.m

```diff
@@ -19,39 +22,59 @@ function convertSourceToRaw(cfg)
     %
     % (C) Copyright 2020 CPP_BIDS developers
 
+    args = inputParser;
+
+    default_filter = struct([]);
+
+    args.addRequired('cfg', @isstruct);
+    args.addParameter('filter', default_filter, @isstruct);
+
+    args.parse(varargin{:});
+
+    cfg = args.Results.cfg;
+    filter = args.Results.filter;
+
     sourceDir = fullfile(cfg.dir.output, 'source');
     rawDir = fullfile(cfg.dir.output, 'raw');
 
-    % add dummy README and CHANGE file
-    templateFolder = fullfile(fileparts(mfilename('fullpath')), '..', 'templates');
+    % trick use bids matlab copy dataset function
+    bids.copy_to_derivative(sourceDir, ...
```

Comment:
reuse the bids matlab "copy dataset" function as it was literally made for this.